index.js

```diff
@@ -17,7 +17,11 @@ module.exports = (css, settings) => {
 
   processedCss = result.stdout
 
-  if (processedCss instanceof Error || processedCss.name === 'CssSyntaxError') {
+  if (result.stderr) {
+    throw new Error(`postcss failed with ${result.stderr}`)
```

Comment:
Would it make sense to append a paragraph with a link to https://err.sh/next.js/postcss-shape ?

Reply:
Totally! Done